### DISCUSSION TITLE
First time customer 5% off

### DISCUSSION THREAD

**Discussion by maduranga001** [openstore-ecommerce/OpenStore#277] [Q&A]
Hi

Is there is an option to give 5% off for the first time customer? If yes, how?
I also want to prompt 'First time customer 5% off' when browsing the website.

Please, can you advise on this?

Thank you!

**Comment by leedavi** [comment]
Sorry for the late reply.  You can do this with a promotional code.  In the Admin Panel you'll find an option for it.

https://manual.openstore-ecommerce.com/en-gb/#IAQmHCh7ha
